### ISSUE TITLE
Unable to check for PDF's because file extension is not present in the Angular Apps Url

### ISSUE THREAD

**Issue by Metroidaron** [react-native-community/react-native-webview#1430]
**TLDR**
How can I access the content type of the Http Request so that I can handle .pdf files for example without depending on the URL; I have come across a website where .pdf (nor any file extension) appears in the URL (This is an Angular Site if that information is helpful at all).

**Is your feature request related to a problem? If so, Please describe.**
When navigating to a PDF document on Android, my application crashes.  I have handled pdf files on other sites using the URL Check; 
- This Url check is in place on the site that crashes my application.  
- This Url Check does work on other sites with PDF files.

**Describe the solutions you came up with**
Is there a way to check the Http Request Content-Type value to check for specific file types such as pdf's? Could this be included in the onLoadStart callback?  Is there another approach that I'm not aware of?  All my google-fu leads to me articles describing how to check the URL for pdf's; but nothing guiding me on when .pdf is not present in the URL.

**Additional context**
Stack Overflow Post: https://stackoverflow.com/questions/62092561/react-native-webview-pdf-file-crashes-application
*I'm posting here for visibility with this community

I am running an outdated version of React-Native-Webview (^6.11.1) but I did update to the latest and still saw this issue with no configuration changes.
